File: backend/src/main/java/ch/zuehlke/fullstack/hackathon/model/Game.java
package ch.zuehlke.fullstack.hackathon.model;


import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import java.io.Serializable;
import java.util.Date;

@Data
@Entity
@Table(name = "game")
public class Game implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private int pos;
    private long time;
    private Date date;
    private String fullName;
    private long pictureId;
    private byte[] picture;
    private String username;
}

File: frontend/src/app/models/game.ts
export default class Game {

  id:number;
  fullName:string;
  pos:number;
  time:number;
  date:Date;
  picture: any;
  pictureId: number;


  constructor(id: number, fullName: string, pos: number, time: number, date: Date, picture: any, pictureId: number) {
    this.id = id;
    this.fullName = fullName;
    this.pos = pos;
    this.time = time;
    this.date = date;
    this.picture = picture;
    this.pictureId = pictureId;
  }
}
